src/middleware/authentication.ts

```diff
@@ -74,7 +74,7 @@ async function decodeAuthChain(req: Request): Promise<string> {
       errorMessage = 'Missing ETH address in auth chain'
     } else {
       try {
-        const endpoint = (req.method + ':' + req.url).toLowerCase()
+        const endpoint = (req.method + ':' + req.path).toLowerCase()
```

Comment:
Do we have any kind of test that verifies this change?

Reply:
We don't actually but I felt like testing the entire authentication middleware was a bit overkill for this PR